auction-server/src/opportunity/service/mod.rs

```diff
@@ -63,6 +63,7 @@ pub struct ConfigSvm {
     pub wallet_program_router_account: Pubkey,
 }
 
+#[allow(dead_code)]
```

Comment:
this feels kinda gross to drop in selectively, is there a CL arg we can set or something to ignore deadcode lints?

Reply:
agreed but I think it's still useful to have it and treat these as exceptions to the rule. I don't expect many more to appear in the next few months and these were sort of accumulated and ignored from the inception of the project.